app/javascript/dashboard/routes/dashboard/settings/inbox/components/AddCampaign.vue

```diff
@@ -157,18 +157,14 @@ export default {
 
   computed: {
     ...mapGetters({
-      agentList: 'agents/getAgents',
+      senderList: 'agents/getAgents',
```

Comment:
Shouldn't we check for the access here? The agents who have access to the inbox should only be listed as senders, right?

Reply:
Yes